stripe_official.php

```diff
@@ -1445,6 +1445,11 @@ public function getPaymentMethods()
         $query->where('name LIKE "STRIPE_PAYMENT%"');
         $query->where('value = "on"');
 
+        if (Shop::isFeatureActive() === true) {
```

Comment:
In case of new PrestaShop with only one shop, ps_configuration has an id_shop and id_shop_group = null


You can have a monoshop and Shop::isFeatureActive() true.

I think we should get configuration via the object Configuration
Configuration::getValues([Stripe_Official::PAYMENT_VISA, Stripe_Official::PAYMENT_MASTERCARD, ...]);

This is the proper way.